src/lib/components/data-table/DataTableActions.svelte

```
@@ -0,0 +1,99 @@
+					</DropdownMenu.Item>
+				</a>
+			{:else if action.action}
+				<DropdownMenu.Item onclick={action.action}>
+					{@const IconComponent = getIcon(action.icon)}
+					{#if IconComponent}
+						<IconComponent />
+					{/if}
+					{action.label}
+				</DropdownMenu.Item>
+			{/if}
+		{/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)